birdy/client/utils.py

```diff
@@ -111,6 +112,8 @@ def convert_output_value(value, data_type):
         value = dateutil.parser.parse(value).time()
     elif "date" in data_type:
         value = dateutil.parser.parse(value).date()
+    elif "angle" in data_type:
```

Comment:
Why is it `angle`? Shouldn't it be `float`?
```
//www.w3.org/TR/xmlschema-2/#float
```

Reply:
https://pywps.readthedocs.io/en/master/_modules/pywps/inout/literaltypes.html

Reply:
ok.